lib/utils/combineJSON.js

```diff
@@ -33,7 +33,7 @@ import { detectDtcgSyntax } from './detectDtcgSyntax.js';
 function traverseObj(obj, fn) {
   for (let key in obj) {
     const prop = obj[key];
-    if (prop) {
+    if (prop !== null && prop !== undefined) {
```

Comment:
```suggestion
    if (prop != null) {
```
Single equals null matches both `null` and `undefined`, so it's a shorthand for this.
ESLint also allows this specific instance of doing single equals vs double equals.

Reply:
Yup thanks, added this in latest commit.